managementApp/src/main/java/com/skytask/controller/ProductController.java

```diff
@@ -1,32 +1,30 @@
 package com.skytask.controller;
 
-import com.skytask.channel.ProductSource;
 import com.skytask.common.Product;
-import com.skytask.service.ProductService;
-import org.springframework.cloud.sleuth.Tracer;
-import org.springframework.integration.support.MessageBuilder;
+import com.skytask.service.RabbitmqService;
+import com.skytask.store.ProductStore;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.servlet.ModelAndView;
 
+import java.util.concurrent.ExecutionException;
+
 @Controller
 class ProductController {
 
-    private ProductService productService;
-    private ProductSource productSource;
-    private Tracer tracer;
+    private ProductStore productStore;
+    private RabbitmqService rabbitmqService;
```

Comment:
This name say to me that your ProductController only works with RabbitMQ, is that what you want?

Reply:
In this specific scenario yes, I am only interacting with rabbitmqService, which sends the messages to the queue.